app/views/idv/finance/new.html.slim

```diff
@@ -0,0 +1,15 @@
+- title t('idv.titles.welcome')
+
+.mt1.mb3.h6.caps.red = t('idv.titles.welcome')
+h2.heading = t('idv.titles.session.finance')
+p.italic Please provide only one of the following:
+= simple_form_for(idv_finance_form, url: idv_finance_path,
+    html: { autocomplete: 'off', method: 'put', role: 'form' }) do |f|
+  = f.error_notification
+  = f.collection_radio_buttons :finance_type, Idv::FinanceForm.finance_type_choices,
+    :first, :last, required: true, item_wrapper_tag: :div
```

Comment:
The format/style of the finance form is still in flux. See https://github.com/18F/identity-private/issues/705#issuecomment-243254350 for example. The form *model* should be the same, though: a single text field. We'll have to apply client-side validation dynamically if we are going to enforce length or patterns. @brendansudol is aware of this.